src/config.js

```diff
@@ -36,7 +36,7 @@ export const urls = {
   thirdPartyFrameRegex: thirdPartyFrameRegex || /^d-\d+\.ampproject\.net$/,
   cdn: env['cdnUrl'] || 'https://cdn.ampproject.org',
   cdnProxyRegex: cdnProxyRegex ||
-      /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org/,
+      /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org(\:\d+)?$/,
```

Comment:
I don't think anchoring it in the end is right. Instead we expect either `:` or `/` and then don't care.

Reply:
This is checked against origins, not URLs.

Reply:
Agreed, this is (currently) only ever checked against url.origin, which can not contain the '/' after the hostname, it's just proto://host[:port]
Happy to add a unit test, pending resolution of the comment thread on the file which I changed+reverted.

Reply:
In that case it works for me. Maybe note that as a comment.

Reply:
Thanks, will do.